.github/workflows/CI.yml

```diff
@@ -3,43 +3,63 @@ name: CI
 on: [pull_request, push]
 
 jobs:
-  build:
+  build-pdf:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/setup-python@v2
+    - name: Run Tune
+      uses: abbbi/github-actions-tune@v1
```

Comment:
We should probably vet this to make sure it has enough users / won't break under us. Otherwise we can fork it like we do for the macOS notarization actions.